pages/linux/plasma-apply-cursortheme.md

```diff
@@ -0,0 +1,16 @@
+# plasma-apply-cursortheme
+
+> Switch Plasma cursor theme.
+> More information: <https://invent.kde.org/plasma/plasma-workspace>.
+
+- List available cursor themes:
+
+`plasma-apply-cursortheme --list-themes`
+
+- Apply a cursor theme:
+
+`plasma-apply-cursortheme {{Cursor_Theme}}`
```

Comment:
```suggestion
`plasma-apply-cursortheme {{CursorTheme}}`
```

Like in plasma-apply-colorscheme?

Reply:
Ah sorry, did not read your previous comment. Maybe we could insert the names of acutal themes/cusors here?

Reply:
Alright, I can agree with that. It would help the user to understand what to input when the same text can be found in the listing.